projects/storefrontstyles/scss/theme/sparta/_fonts.scss

```diff
@@ -1 +1,9 @@
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700');
+@include forVersion(4, 4.99) {
```

Comment:
It's `forVersion(4, 4.99)` for purpose? Can't we use `forVersion(4)`?

Reply:
`forVersion(4.1)` doesn't work like that unfortunately. Changing the versions to `5` or `5.0` still also include the `forVersion(4.1)` function even though comments in `versioning.scss` claim it shouldn't include it in the next major.